x-pack/plugins/infra/public/types.ts

```diff
@@ -4,16 +4,20 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { CoreSetup, CoreStart, Plugin as PluginClass } from 'kibana/public';
-import { DataPublicPluginStart } from '../../../../src/plugins/data/public';
-import { HomePublicPluginSetup } from '../../../../src/plugins/home/public';
-import {
+import type { CoreSetup, CoreStart, Plugin as PluginClass } from 'kibana/public';
```

Comment:
This just tries to make sure we don't accidentally pull runtime code into the eagerly loaded entry point bundle.